docs/service-mesh/10-03-connection-refused.md

```diff
@@ -3,7 +3,9 @@ title: Connection refused errors after upgrade
 type: Troubleshooting
```

Comment:
Nit, maybe change title? It was valid for kyma 1.0. Such errors may now be not only after upgrade but also after fresh installation